src/ddb/ddb_commands.c

```diff
@@ -1,5 +1,5 @@
 /**
- * (C) Copyright 2022 Intel Corporation.
+ * (C) Copyright 2023 Intel Corporation.
```

Comment:
It probably doesn't matter since this will all be squashed into a single merge, but generally the copyright update shouldn't replace the year with the current year, it should append a range, e.g. (2022-2023). Are you using the git commit hooks? This should be handled automatically for you, fwiw. In any case, I wouldn't block on this, just calling it out for future reference.

Reply:
Good point.  We probably need to update the commit hooks at some point.